frontend/my-app/.gitignore

```diff
@@ -14,6 +14,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
i'm not sure this needs to be added to the `.gitignore`, as it is not a local file

Reply:
If I don't add it it shows up as wanting to track it. I'd prefer to leave it so it doesn't pop up every time but don't feel too strongly

Reply:
Isn't this a good thing? Anyone who runs the application is going to need to hit the backend, plus if we need more environmental variables in the future it's good to have it

Reply:
Sorry, I'm confused - what's the good thing here? Including .env in the gitignore file or something else? Or instead including the .env file in the repository?

Reply:
Resolved on slack! Added .env and example backend url